docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md

```diff
@@ -147,13 +147,13 @@ select wf01.wt01.status,wf02.wt02.hardware from root.ln where (time > 2017-11-01
 
 ### 降频聚合查询
```

Comment:
`5-Operation Manual/4-SQL Reference.md` needs updating too.

Reply:
I've updated the corresponding part.